src/calculation/MeasureReportBuilder.ts

```diff
@@ -217,7 +237,11 @@ export default class MeasureReportBuilder<T extends PopulationGroupResult> exten
             er.stratifierResults?.forEach(stratResults => {
               // only add to results if this episode is in the strata
               if (stratResults.result) {
-                const strata = group.stratifier?.find(s => s.code && s.code[0].text === stratResults.strataCode);
+                // the strataCode has the potential to be a couple of things, either s.code[0].text (previous measures)
+                // or s.id (newer measures)
+                const strata: MeasureReportGroupStratifier | undefined =
+                  group.stratifier?.find(s => s.code && s.code[0]?.text === stratResults.strataCode) ||
+                  group.stratifier?.find(s => s.id && s.id === stratResults.strataCode);
```

Comment:
```suggestion
                  group.stratifier?.find(s => s.id === stratResults.strataCode);
```
Could be wrong if there's something weird about the stratResults, but I think that we don't need the extra `s.id` check here.